dashboard/app/models/user.rb

```diff
@@ -76,7 +76,7 @@
 
 class User < ActiveRecord::Base
   include SerializedProperties
-  serialized_attrs %w(ops_first_name ops_last_name district_id ops_school ops_gender)
+  serialized_attrs %w(ops_first_name ops_last_name district_id ops_school ops_gender races race_interstitial_shown)
```

Comment:
Does it make sense to `unknown` or `prefer not to say` in `races` if the student dismisses the interstitial? Doing so could eliminate the need for `race_interstitial shown`.